less/opendev.less

```diff
@@ -12,6 +12,7 @@ img {
 }
 
 table {
```

Comment:
@Huyeng there is actually a problem with using datatables on those kind of tables as the one present on https://pp.opendevelopmentcambodia.net/topics/protected-forest/. Since those tables are created using the WP editor, they might NOT have a header <thead></thead> which determines the table's headers and is actually needed by datatables.

see https://stackoverflow.com/questions/29893207/datatables-typeerror-c-is-undefined